packages/protocol/test/forking/arbitrum/CompoundV3Arbitrum.t.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.15;
+
+import "forge-std/console.sol";
+import {Routines} from "../../utils/Routines.sol";
+import {ForkingSetup} from "../ForkingSetup.sol";
+import {CompoundV3Arbitrum} from "../../../src/providers/arbitrum/CompoundV3Arbitrum.sol";
+import {AaveV3Arbitrum} from "../../../src/providers/arbitrum/AaveV3Arbitrum.sol";
+import {ICompoundV3} from "../../../src/interfaces/compoundV3/ICompoundV3.sol";
+import {ILendingProvider} from "../../../src/interfaces/ILendingProvider.sol";
+import {BorrowingVault} from "../../../src/vaults/borrowing/BorrowingVault.sol";
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import {Chief} from "../../../src/Chief.sol";
+import {TimelockController} from
+  "openzeppelin-contracts/contracts/governance/TimelockController.sol";
+
+contract CompoundV3ArbitrumForkingTest is Routines, ForkingSetup {
```

Comment:
Note that in the latest main branch, all ForkingTest was replaced with ForkingTests with an "s". This was done in order to match the regex pattern for running "UnitTests".